core/domain/exp_services_test.py

```diff
@@ -7078,46 +7039,6 @@ def test_update_exploration_auto_tts_enabled(self) -> None:
         self.assertEqual(exploration.title, 'new title')
         self.assertEqual(exploration.auto_tts_enabled, True)
 
-    def test_update_exploration_correctness_feedback_enabled(self) -> None:
```

Comment:
can you explain a little more on this? we already have test for other properties.

Reply:
This test is not actually trying to test the property. It's trying to see if an older version of the exploration can still be updated (look at the PR which introduced it, using git blame). The author of the test just used the correctness_feedback_enabled field because it was convenient.

The request here is to keep the test but use a different field instead.

Reply:
added test, PTAL.